src/Http/Extra.elm

```diff
@@ -392,3 +402,25 @@ testing
 toSettings : RequestBuilder -> Settings
 toSettings (RequestBuilder request settings) =
   settings
+
+
+{-| Join the arguments of a list of string touples in an url encoded format
+-}
+joinUrlEncoded: List (String, String) -> String
+joinUrlEncoded args =
+  String.join "&" (List.map queryPair args)
+
+
+queryPair : (String,String) -> String
+queryPair (key,value) =
+  queryEscape key ++ "=" ++ queryEscape value
+
+
+queryEscape : String -> String
+queryEscape string =
+  string |> Http.uriEncode |> replace "%20" "+"
+
+
+replace : String -> String -> String -> String
+replace old new string =
+  string |> String.split old |> String.join new
```

Comment:
Stole from the original Evan's code with sligly modifications. I don't really deserve credit on this stuff.